js/app.js

```diff
@@ -0,0 +1,16 @@
+var giphyKey = "bAqrGC0EFBsitN09IxRQsJdQPme35o1E";
```

Comment:
FYI - I know we didn't talk about it much in class, but it's good practice to start using 'let' instead of var. No changes needed though 👍